pyopia/pipeline.py

```diff
@@ -9,6 +9,21 @@ class Pipeline():
     '''The processing pipeline class
     ================================
 
+    The casses called in the Pipeline steps can be modified, and the names of the steps changed.
+    New steps can be added or deleted as required.
+
+    The classes called in the Pipeline steps need to take a dictionary and input and return a dictionary as output.
+    This common dictionary is therefore passed between steps so that data
+    or variables generated by each step can be passed along the pipeline.
+
+    The `steps` dict must contain keys: 'common' and 'classifier',
+    which will run when Pipeline is initialised - the remaining steps will be run on Pipeline.run()
+
+    The step called 'classifier' must return a dict containing 'cl' in order to run successfully.
```

Comment:
Is this still true? I believe it should not be strictly true, just kind of a normal convention to follow now.

Reply:
i think this is still true, but not quite sure what you mean? (this will also be better in the next PR where I have improved this a bit).